javascript/javascript-coding-conventions.md

```diff
@@ -86,6 +86,14 @@ The primary goal is to make the page load as quickly as possible for the user. W
 	</body>
 </html>
 ```
+#### Other trick
+In some cases, if you do not want to move script tag to bottom of page, consider to use async and defer attribute in script tag.
+
+```
+<script src="scripts.js" async></script>
+<script src="scripts.js" defer></script>
```

Comment:
This is interesting. I think you should add some useful reference links to `async` & `defer`.